requirements.txt

```diff
@@ -1,50 +1,50 @@
-aiohttp==3.7.4
-appdirs==1.4.4
-asgiref==3.3.1
-async-timeout==3.0.1
-beautifulsoup4==4.9.3
-boto3==1.16.36
-botocore==1.19.36
-certifi==2020.12.5
-cffi==1.14.4
-chardet==3.0.4
-cryptography==3.3.2
-Django==3.1.6
-django-cognito-jwt==0.0.3
-django-cors-headers==3.5.0
-django-extensions==3.1.0
-django-smtp-ssl==1.0
-djangorestframework==3.12.2
-Faker==5.0.1
-geoip2==4.1.0
-gophish==0.5.1
-gunicorn==20.0.4
-idna==2.10
-jmespath==0.10.0
-maxminddb==2.0.3
-multidict==5.1.0
-packaging==20.8
-pycparser==2.20
-pyee==8.1.0
-PyJWT==1.7.1
-pymongo==3.11.2
-pyparsing==2.4.7
-pyppeteer==0.2.2
-python-dateutil==2.8.1
-pytz==2020.4
-PyYAML==5.4
-regex==2020.11.13
-requests==2.25.0
-s3transfer==0.3.3
-schematics==2.1.0
-simpleeval==0.9.10
-six==1.15.0
-soupsieve==2.0.1
-sqlparse==0.4.1
-text-unidecode==1.3
-tqdm==4.54.1
-ua-parser==0.10.0
-urllib3==1.26.3
-user-agents==2.2.0
-websockets==8.1
-yarl==1.6.3
+aiohttp>=3.7.4
+appdirs>=1.4.4
+asgiref>=3.3.1
+async-timeout>=3.0.1
+beautifulsoup4>=4.9.3
+boto3>=1.16.36
+botocore>=1.19.36
+certifi>=2020.6.20
+cffi>=1.14.4
+chardet>=3.0.4
+cryptography>=3.3.2
+Django>=3.1.6
+django-cognito-jwt>=0.0.3
+django-cors-headers>=3.5.0
+django-extensions>=3.1.0
+django-smtp-ssl>=1.0
+djangorestframework>=3.12.2
+Faker>=5.0.1
```

Comment:
This looks like testing-specific foo that should be in `requirements-test.txt`.

Reply:
@jsf9k The ones that are highlighted are not testing specific. All of those libraries are used in the main application.

Reply:
Thanks!